apax/utils/helpers.py

```diff
@@ -2,22 +2,26 @@
 
 import yaml
 
+APAX_PROPERTIES = [
+    "energy",
+    "forces",
+    "stress",
+    "forces_uncertainty",
+    "energy_uncertainty",
+    "stress_uncertainty",
+    "energy_ensemble",
+    "forces_ensemble",
+    "stress_ensemble",
+]
+
 
 def setup_ase():
     """Add uncertainty keys to ASE all properties.
     from https://github.com/zincware/IPSuite/blob/main/ipsuite/utils/helpers.py#L10
     """
     from ase.calculators.calculator import all_properties
 
-    additional_keys = [
-        "forces_uncertainty",
-        "energy_uncertainty",
-        "stress_uncertainty",
-        "energy_ensemble",
-        "forces_ensemble",
-    ]
-
-    for val in additional_keys:
+    for val in APAX_PROPERTIES:
         if val not in all_properties:
             all_properties.append(val)
```

Comment:
`all_properteis.extend(APAX_PROPERTIES)`

Reply:
Is it a problem if properties occur more than once in ASE's `all_properties`?

Reply:
should not - but probably better to leave it as is then.